src/core/sync/index.ts

```diff
@@ -80,7 +78,7 @@ export function sync<Domain>(params: Params<Domain>) {
   const push = makePush({
     canonData,
     onError: err => {
-      setConnectivity("crashed");
+      connectivity.set("crashed");
```

Comment:
Can we convert these string literals to constants? Doing so makes it eaiser to leverage tooling to find all usages.

Reply:
It comes with its own drawbacks, though, like having to import the constants before use, and needing to make constants tagged objects to prevent accidental literal string use.

The way I find all usages is I change the type and TypeScript lets me know all the existing instances, since they don’t match the type anymore.

Regardless, this is outside the scope of this PR, let’s continue discussing it in sync in chat!